src/components/touchableOpacity/index.tsx

```diff
@@ -48,6 +48,10 @@ export interface TouchableOpacityProps
    * Custom value of any type to pass on to TouchableOpacity and receive back in onPress callback
    */
   customValue?: any;
+  /**
+   * Testing identifier
+   */
+  testID?: string;
```

Comment:
I don't think you don't need to define a testID prop on TouchableOpacity, it already inherit it from React Native's TouchableOpacity

Reply:
You're right. Fixed.